src/e2e/scala/io/github/paoloboni/E2ETests.scala

```diff
@@ -52,15 +55,15 @@ class E2ETests extends AsyncFreeSpec with AsyncIOSpec with Matchers with Env {
           )
         )
       )
-      .asserting(_ shouldBe a[binance.OrderId])
+      .asserting(_ shouldBe a[OrderId])
   }
 
   "cancelOrder" in {
     BinanceClient[IO](config)
       .use(client =>
         for {
           id <- client.createOrder(
-            OrderCreate(
+            OrderCreationParameters(
```

Comment:
I see we are repeating the work Parameters in all those definitions, what do you think about namespacing these classes by using an object, such as:
```
object Parameters {
   case class OrderCreation ...
}
```
And then we could reference these as
 ```suggestion
             Parameters.OrderCreation(
```
what do you think?

Reply:
When we want to add more and more of the api we would get one monolitic `object Parameters` again.
Currently they are in the `parameter` package, we could refer to them as `parameters.OrderCreation`